CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.6.3] - 2023-01-13
+### Fixed
+ - Added the 'nobody' user to the image.
```

Comment:
Bigger change is switching to the algol60 base image to resolve CVE's.

Reply:
Fixed.